test/Unit/lit.cfg

```diff
@@ -77,3 +78,10 @@ config.test_format = lit.formats.GoogleTest(config.build_mode, 'Tests')
 # module cache for all those tests that do not set that explicitly
 if 'XDG_CACHE_HOME' in os.environ:
   config.environment['XDG_CACHE_HOME'] = os.environ['XDG_CACHE_HOME']
+
+lit_values = {}
+with open(os.path.join(config.lit_site_config_folder, "..", "lit_values.json"), 'r') as lit_values_file:
+    lit_values = json.load(lit_values_file)
+if 'use_os_stdlib' not in lit_config.params and 'target_stdlib_path' in lit_values:
+   config.environment['DYLD_LIBRARY_PATH'] = lit_values['target_stdlib_path']
```

Comment:
Very likely `SIMCTL_CHILD_DYLD_LIBRARY_PATH` needs to be set as well, but currently not sure how.

Reply:
Just realized these tests only run for the target platform, so we can simplify this logic.